experimental/ComplexReflectionGroups/src/complex_reflection_group_CHEVIE.jl

```diff
@@ -0,0 +1,609 @@
+
+###########################################################################################
+# Models from CHEVIE
+###########################################################################################
+function complex_reflection_group_CHEVIE(n::Int)
+
+    if n == 4
+        K,z_3 = cyclotomic_field(3)
+        matspace = matrix_space(K, 2, 2)
+        gens = elem_type(matspace)[]
+
+        s1 = matspace(K[1 0 ; 0 z_3])
+        s2 = matspace(K[1//3*(2*z_3 + 1) 1//3*(2*z_3 - 2) ; 1//3*(z_3 - 1) 1//3*(z_3 + 2)])
```

Comment:
```suggestion
        gens = dense_matrix_type(K)[]

        s1 = K[1 0 ; 0 z_3]
        s2 = K[1//3*(2*z_3 + 1) 1//3*(2*z_3 - 2) ; 1//3*(z_3 - 1) 1//3*(z_3 + 2)]
```
Using the `matrix_space` is redundant here; `K[1 0 ; 0 1]` already gives an OSCAR matrix and the field knows about the dense matrix type itself. More cases below, I'm afraid.